### ISSUE TITLE
GATK 4.1.2.0 - CollectOxoGMetrics - GC overhead limit exceeded

### ISSUE THREAD

**Issue by paulstretenowich** [broadinstitute/gatk#6907]
Hi,

I'm using CollectOxoGMetrics with GATK 4.1.2.0 and I'm having a `GC overhead limit exceeded` issue on some samples I have.
The command line used is:
`gatk --java-options "-XX:ParallelGCThreads=15 -Xmx60G"   CollectOxoGMetrics --VALIDATION_STRINGENCY=SILENT --INPUT=input.bam --OUTPUT=oxog_metrics.txt --REFERENCE_SEQUENCE=reference.fa --MAX_RECORDS_IN_RAM=1000000`

I tried first with `-XX:ParallelGCThreads=1`, `-Xmx6G` and `--MAX_RECORDS_IN_RAM=4000000` but I ended with the same `GC overhead limit exceeded` issue.
I also modified the previous parameters mainly increasing ParallelGCThreads and memory and decreasing MAX_RECORDS_IN_RAM but nothing changed.

Don't hesitate if you need more information.

Thanks for the help,

Paul

**Comment by droazen** [comment]
@paulstretenowich Can you try running with the latest release (4.1.9.0) and see if you get the same error?

**Comment by paulstretenowich** [comment]
Hi @droazen, I tried also with the latest version and I had the same issue. I tried over the week end with `-XX:ParallelGCThreads=20`, `-Xmx60G` and `--MAX_RECORDS_IN_RAM=400000` but it ended after 37h of run with `Exception in thread "main" java.lang.OutOfMemoryError: Java heap space: failed reallocation of scalar replaced objects`.
On more than 95% of the samples I have it's working and taking less than 1h.
